server/listeners/eventHandlers/prisonEvents/datesChangedEventHandler.ts

```diff
@@ -39,18 +46,22 @@ export default class DatesChangedEventHandler {
     }
   }
 
-  deactivateLicenceIfPrisonerResentenced = async (licence: LicenceSummary, prisoner: PrisonApiPrisoner) => {
+  deactivateLicenceIfPrisonerResentenced = async (licences: LicenceSummary[], prisoner: PrisonApiPrisoner) => {
```

Comment:
I think this should be plural too (ie `deactivateLicences` rather than `deactivateLicence`)

Reply:
changed to deactivateLicences.